shared/pingpong/accounts.go

```diff
@@ -186,133 +194,156 @@ func prepareAssets(accounts map[string]uint64, client libgoal.Client, cfg PpConf
 			totalSent++
 			throttleTransactionRate(startTime, cfg, totalSent)
 		}
-		account, accountErr := client.AccountInformation(addr)
-		if accountErr != nil {
-			fmt.Printf("Cannot lookup source account %v\n", addr)
-			err = accountErr
-			return
-		}
+	}
 
+	// wait until all the assets created
+	allAssets := make(map[uint64]string, cfg.NumAsset)
```

Comment:
That's an interesting way of doing it. I'm not sure it's the way I would do that - 
I think it would be more robust to send a transaction and verify that my transaction was included in a block. This way, there is no need to poke into the account. ( i.e. the account could have modified by a another transaction which wasn't sent in this context ).

Reply:
there are too many transactions to store IDS and lookup in blocks. Plus, we need to know asset id that does not appear in txn we are looking for (it is a tx counter but it is implementation details).